### PR TITLE
Add org.cubocore.CoreImage

### PR DESCRIPTION
**Pull request by s96Abrar**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreImage

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreImage

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreImage

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreImage

File: org.cubocore.CoreImage.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
```
Comment (barthalion) [inline org.cubocore.CoreImage.yml:9]:
```suggestion
  - --socket=fallback-x11
```

File: org.cubocore.CoreImage.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.cubocore.CoreImage.appdata.xml:1]: You should consider maintaining this file in your git repo as it's not specific to Flathub.

File: org.cubocore.CoreImage.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --share=network
```
Comment (barthalion) [inline org.cubocore.CoreImage.yml:8]: What does it need network access for?

File: org.cubocore.CoreImage.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
```
Comment (barthalion) [inline org.cubocore.CoreImage.yml:11]: What does it need PulseAudio access for?

File: org.cubocore.CoreImage.appdata.xml (hunk 2 above)
Comment (s96Abrar) [reply org.cubocore.CoreImage.appdata.xml:1]: I will maintain it in git. Thanks for the pointing.

File: org.cubocore.CoreImage.yml (hunk 3 above)
Comment (s96Abrar) [reply org.cubocore.CoreImage.yml:8]: My bad accidentally added. It does not require any network.

File: org.cubocore.CoreImage.yml (hunk 4 above)
Comment (s96Abrar) [reply org.cubocore.CoreImage.yml:11]: Again, my bad accidentally added. It does not require any pulseaudio socket.

File: org.cubocore.CoreImage.yml (hunk 3 above)
Comment (s96Abrar) [reply org.cubocore.CoreImage.yml:8]: The app didn't require any network though. But having little warning with `network`
`Qt: Session management error: Could not open network socket`

Will it be a problem if `network` used?

**Comment by nedrichards** [comment]
bot, build org.cubocore.CoreImage

File: org.cubocore.CoreImage.yml (hunk 3 above)
Comment (barthalion) [reply org.cubocore.CoreImage.yml:8]: No, I believe it's about something else.

Comment (s96Abrar) [reply org.cubocore.CoreImage.yml:8]: The warning you mean?

Comment (barthalion) [reply org.cubocore.CoreImage.yml:8]: Yeah, I don't remember the details anymore. It's not critical anyway, should be a warning.

Comment (s96Abrar) [reply org.cubocore.CoreImage.yml:8]: OK Fine.

**Comment by nedrichards** [comment]
bot, build org.cubocore.CoreImage

File: org.cubocore.CoreImage.appdata.xml (hunk 2 above)
Comment (s96Abrar) [reply org.cubocore.CoreImage.appdata.xml:1]: Will it be a matter to store the `appdata.xml` to github/flathub repo for now until the app got another release?

File: org.cubocore.CoreImage.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```
Comment (nedrichards) [inline org.cubocore.CoreImage.yml:10]: We need a more detailed justification for the filesystem=host permission. xdg-pictures would be a more appropriate default and have fewer security concerns.

Comment (s96Abrar) [reply org.cubocore.CoreImage.yml]: Updated to `xdg-pictures`

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreImage

**Pull request closed without merging**